app/serializers/request_serializer.rb

```diff
@@ -1,4 +1,21 @@
 class RequestSerializer
   include JSONAPI::Serializer
-  attributes :name, :details, :item_category, :status, :borrow_date, :return_date, :requested_by_id, :lender_id, :community_id
+  attributes :name, :details, :item_category, :status, :requested_by_id, :lender_id, :community_id, :created_at, :updated_at
+
+  attributes :borrow_date do |object|
+    object.borrow_date.strftime("%m-%d-%y")
+  end
+
+  attributes :return_date do |object|
+    object.return_date.strftime("%m-%d-%y")
```

Comment:
Dallas mentioned that Date objects should be renamed to follow Rails convention...
date objects should follow (borrow_on / return_on)
datetime objects should follow (created_at / updated_at / borrow_at)

Reply:
Nice catch, my branch must have been out of date when I merged and it had some of your all's old code.